scripts/verify-agent-artifacts

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 AGENT_VERSION="${1:-}"
+INIT_VERSION="${AGENT_VERSION}-1"
```

Comment:
Would it always end with `-1`? I found that we have [Amazon ECS Init - v1.58.0-2](https://github.com/aws/amazon-ecs-init/releases/tag/v1.58.0-1) and  [Amazon ECS Init - v1.52.2-2](https://github.com/aws/amazon-ecs-init/releases/tag/v1.52.2-2) released in the previous.

Reply:
It will always end with `-1` now that we have init and agent in the same repo.